whenchanged/whenchanged.py

```diff
@@ -191,7 +209,17 @@ def main():
         if verbose:
             print("When '%s' changes, run '%s'" % (files[0], print_command))
 
-    wc = WhenChanged(files, command, recursive, run_once, run_at_start)
+    return WhenChanged(files, command, recursive, run_once, run_at_start)
```

Comment:
Kwarg seems good, I thought returning an object was more straightforward (especially considering the fields are all public in Python), but adding an indirection is easy to do.